mcs/class/corlib/System/TermInfoReader.cs

```diff
@@ -114,12 +116,21 @@ public TermInfoReader (string term, byte [] buffer)
 //			get { return term; }
 //		}
 
+		void DetermineVersion (short magic)
+		{
+			if (magic == 0x11a)
+				intOffset = 2;
+			else if (magic == 0x21e)
+				intOffset = 4;
+			else
+				throw new Exception (String.Format ("Magic number is wrong: {0}", magic));
```

Comment:
maybe s/wrong/unexpected ?

Reply:
I kept the old message, but I agree, the wording feels odd. Will fix.